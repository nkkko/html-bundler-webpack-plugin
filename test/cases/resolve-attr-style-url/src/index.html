<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <h1>Hello World!</h1>
    <div style="color: red;">text</div>
    <div style="width: 160px; background-image: url(@images/image.webp); height: 130px;"></div>
    <div style="width: 180px; background-image: url('@images/2k-over/bild.png'); height: 140px;"></div>
  </body>
</html>