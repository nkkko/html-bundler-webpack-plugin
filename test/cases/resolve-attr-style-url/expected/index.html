<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  </head>
  <body>
    <h1>Hello World!</h1>
    <div style="color: red;">text</div>
    <div style="width: 160px; background-image: url(img/image.d4711676.webp); height: 130px;"></div>
    <div style="width: 180px; background-image: url('img/bild.7b396424.png'); height: 140px;"></div>
  </body>
</html>